remy/remarkable/filesource.py

```diff
@@ -467,22 +483,20 @@ def fileSourceFromSSH(cls, name="SSH", address='10.11.99.1', username='root', pa
         else:
           raise Exception("A passphrase for SSH key is required")
     else:
-      pkey = None
-      if password is None:
-        raise Exception("Must provide either password or SSH key")
+      # as a last option we can try to let paramiko handle the keys
+      pkey=None
 
     options = {
       'username': username,
       'password': password,
       'pkey': pkey,
       'timeout': timeout,
-      'look_for_keys': False
+      'look_for_keys': pkey is not None
```

Comment:
Shouldn't this be `'look_for_keys': pkey is None`?

Reply:
fixed it